tests/v2/validation/scale/scale_etcd_down_then_up_test.go

```diff
@@ -0,0 +1,78 @@
+package scale
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/rancher/tests/v2/validation/extensions/provisioning/crud"
+	"github.com/rancher/rancher/tests/v2/validation/extensions/provisioning/permutations"
+	"github.com/rancher/rancher/tests/v2/validation/provisioning"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type EtcdNodeScaleDownAndUp struct {
+	suite.Suite
+	session        *session.Session
+	client         *rancher.Client
+	ns             string
+	clustersConfig *provisioning.Config
+}
+
+func (s *EtcdNodeScaleDownAndUp) TearDownSuite() {
+	s.session.Cleanup()
+}
+
+func (s *EtcdNodeScaleDownAndUp) SetupSuite() {
+	testSession := session.NewSession()
+	s.session = testSession
+
+	s.ns = provisioning.Namespace
+
+	s.clustersConfig = new(provisioning.Config)
+	config.LoadConfig(provisioning.ConfigurationFileKey, s.clustersConfig)
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(s.T(), err)
+
+	s.client = client
+}
+
+func (s *EtcdNodeScaleDownAndUp) TestEtcdNodeScaleDownAndUp() {
+	clusterNames := s.client.RancherConfig.ClusterNames
+	if len(clusterNames) == 0 {
```

Comment:
I don't like this at all, either we just read from the config a single cluster name. Or we can move this back into provision, and provision each cluster type, and validate accordingly which would be even simpler since the Audit or validation is already an extension.

Reply:
okay I can rip this out and move the functions below back to provisioning.

Reply:
ripping this out